assignment7/deleteClient.js

```diff
@@ -1,22 +1,22 @@
-var express = require("express");
+var express = require('express');
 var router = express.Router();
-var requirePromise = require("request-promise");
+var requirePromise = require('request-promise');
 
-router.get("/:idClient", function(req, res) {
+router.get('/:idClient', function(req, res) {
   var idClient = req.params.idClient;
-  var url = "https://api.createsend.com/api/v3.1/clients/" + idClient + ".json";
+  var url = 'https://api.createsend.com/api/v3.1/clients/' + idClient + '.json';
   requirePromise
     .delete(url, {
       auth: {
-        user: "fc96dc9ded698bbfbfb975409a864a42",
-        pass: "x"
+        user: 'fc96dc9ded698bbfbfb975409a864a42',
+        pass: 'x'
       }
     })
     .then(function(response) {
-      res.send("Client with client ID : " + idClient + " deleted");
+      res.send('Client with client ID : ' + idClient + ' deleted');
     })
     .catch(function(err) {
-      console.log("error");
+      console.log('error');
```

Comment:
@abansal26 Use err in console message i.e console.log('error',err), so you can easily debug the error cause.